### PR TITLE
Polishing Vagrant workflow

### PR DESCRIPTION
**Pull request by techtonik**
This updates README.md, removes some warning and fills in missing dependencies

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -244,15 +244,23 @@ development then sign up for an account of your own at
 
 Vagrant
 -------
-If you have Vagrant installed, you can run Gratipay by running `vagrant up` from the project directory. Please note that if you ever switch between running Gratipay on your own machine to Vagrant or vice versa, you will need to run `make clean`.
+Vagrant provides a convenient interface to VirtualBox to run and test
+Gratipay in virtual machine. This may be handy if you're on Windows.
 
-If you're using Vagrant for the first time you'll need [Vagrant](http://www.vagrantup.com/) and [VirtualBox](https://www.virtualbox.org/) installed. If you're on Linux you'll need to install `nfs-kernel-server`.
+You will [Vagrant](http://www.vagrantup.com/) and [VirtualBox](https://www.virtualbox.org/)
```
Comment (rohitpaulk) [inline README.md]: This should read "You will **need**"

File: README.md [hunk 2]
```diff
@@ -244,15 +244,23 @@ development then sign up for an account of your own at
...
+installed. On Linux you may need to install `nfs-kernel-server` as well.
 
-The `Vagrantfile` will download a custom made image from the Internet. If you have a slow internet connection, you can download a local copy of this file, by running:
+With Vagrant, you can run Gratipay by running `vagrant up` from the project
+directory. Please note that if you switch between running Gratipay on your
+own machine to Vagrant and vice versa, you need to run `make clean`.
```
Comment (rohitpaulk) [inline README.md]: Though these are small grammatical changes, IMO the previous version was better - 

> Please note that if you ever switch between running Gratipay on your own machine to Vagrant or vice versa, you will need to run `make clean`.

**Comment by rohitpaulk** [comment]
Hey @techtonik :) 

I've left a few comments on the changes to `README.md`.

File: README.md (hunk 1 above)
Comment (techtonik) [inline README.md]: Fixed!

File: README.md (hunk 2 above)
Comment (techtonik) [inline README.md]: Sometimes it happens. Fixed. =) At first I thought that the English was too complicated, but from the other side - how would people learn?

**Comment by techtonik** [comment]
Cool to know that your English is reviewed as well. =)

I've also added condition to choose 32-bit image to speed up VirtualBox on 32-bit systems. I don't see a drastic improvement in performance, but at least I have a feeling that it should be faster.

**Comment by techtonik** [comment]
!m @whit537 @rohitpaulk =)

The Vagrant/VirtualBox setup is still horribly slow on my 32-bit Vista. I will probably switch to Codenvy setup, which allows to run Docker container and test changes from the web.

**Pull request merged**